src/ticks.js

```diff
@@ -0,0 +1,75 @@
+import {bisector, tickStep} from "d3-array";
```

Comment:
Not sure if this is worth adding the dependency between libs even though d3-array does seem to be more fundamental and therefore possibly useful in time.